### PR TITLE
NAS-100551 Update managing jail docs to match current UI proccess.

### PR DESCRIPTION
**Pull request by a-stjohn**

### CODE REVIEW

File: userguide/jails.rst [hunk 1]
```diff
@@ -816,44 +771,45 @@ available for a jail.
    | Option       | Description                                                   |
    |              |                                                               |
    +==============+===============================================================+
-   | Edit         | Used to modify the settings described in                      |
-   |              | :numref:`Table %s <jail_overview_tab>`.                       |
+   | EDIT         | Used to modify the settings described in                      |
+   |              | :ref:`Advanced Jail Creation`.                                |
    |              | A jail cannot be edited while it is running. The settings can |
```
Comment (wblock) [inline userguide/jails.rst]: Duplicate "can".

File: userguide/jails.rst [hunk 2]
```diff
@@ -900,12 +856,12 @@ The ssh daemon
...
-The jail :guilabel:`Status` must be up before the :guilabel:`Shell`
+The jail :guilabel:`State` must be *up* before the :guilabel:`SHELL`
 option is available. If the jail is not up, start it by clicking
-:menuselection:`Jails -->` |ui-options| :menuselection:`--> Start`
+:menuselection:`Jails -->` |ui-chevron-right| :menuselection:`--> START`
 for the desired jail. Click
-|ui-options| :menuselection:`--> Shell`
-to start a shell on the jail. A jail root shell is shown in this
+|ui-chevron-right| :menuselection:`--> SHELL`
+to open the shell of the jail. A jail root shell is shown in this
```
Comment (wblock) [inline userguide/jails.rst]:
```suggestion
to open a shell in the jail. A jail root shell is shown in this
```

File: userguide/jails.rst [hunk 3]
```diff
@@ -1051,18 +1007,19 @@ storage area within a jail.
 The :guilabel:`Mount points` section of a jail shows any added storage
```
Comment (wblock) [inline userguide/jails.rst]: Is this the correct case?

File: userguide/jails.rst [hunk 4]
```diff
@@ -733,58 +733,16 @@ example is shown in :numref:`Figure %s <jail_overview_fig>`.
...
-
-
 Operations can be applied to multiple jails by selecting those jails
 with the checkboxes on the left. After selecting one or more jails,
 icons appear which can be used to |ui-jail-start|, |ui-jail-stop|,
 |ui-jail-update|, or |ui-jail-delete| those jails.
 
-Click |ui-options| for a jail to see all options for that jail.
+Click |ui-chevron-right| for a jail to see the *IPV4*, *IPV6*, *TYPE*
+of jail, if it is a *TEMPLATE* jail or not, and if it is a *BASEJAIL*
```
Comment (wblock) [inline userguide/jails.rst]: The "or not" is rarely needed.
```suggestion
of jail, whether it is a *TEMPLATE* jail, and whether it is a *BASEJAIL*.
```

File: userguide/jails.rst [hunk 5]
```diff
@@ -733,58 +733,16 @@ example is shown in :numref:`Figure %s <jail_overview_fig>`.
...
+or not. It also displays additional options for that jail which are
```
Comment (wblock) [inline userguide/jails.rst]:
```suggestion
It also displays additional options for that jail which are 
```

File: userguide/jails.rst [hunk 6]
```diff
@@ -817,44 +772,45 @@ available for a jail.
...
+   |              | :ref:`Advanced Jail Creation`.                                |
+   |              | A jail cannot be edited while it is running. The settings     |
    |              | can be viewed, but are read only.                             |
    |              |                                                               |
    +--------------+---------------------------------------------------------------+
-   | Mount        | Open the :guilabel:`Mount Points` list. Select an existing    |
-   | points       | mount point to :guilabel:`Edit` or click |ui-add| to open     |
-   |              | the :guilabel:`Add Mount Point` screen. A mount point         |
+   | MOUNT        | Select an existing                                            |
+   | POINTS       | mount point to :guilabel:`EDIT` or click                      |
```
Comment (wblock) [inline userguide/jails.rst]: Is "mount points" always two words, or should it be a compound word (mountpoints)?

File: userguide/jails.rst [hunk 7]
```diff
@@ -817,44 +772,45 @@ available for a jail.
...
+   |              | :menuselection:`ACTIONS --> Add Mount Point`                  |
+   |              | to create a mount point for the jail. A mount point           |
    |              | gives a jail access to storage located elsewhere on the       |
    |              | system. A jail must be stopped before adding, editing, or     |
-   |              | deleting a :guilabel:`Mount Point`. See                       |
+   |              | deleting a :guilabel:`MOUNT POINT`. See                       |
```
Comment (wblock) [inline userguide/jails.rst]: Not sure why this one is a guilabel when the previous references are just generic text.  This one probably should be plain text also.

File: userguide/jails.rst [hunk 8]
```diff
@@ -817,44 +772,45 @@ available for a jail.
...
    +--------------+---------------------------------------------------------------+
-   | Stop         | Stop a jail that has a current :guilabel:`Status` of          |
+   | STOP         | Stop a jail that has a current :guilabel:`STATE` of           |
    |              | *up*.                                                         |
    |              |                                                               |
    +--------------+---------------------------------------------------------------+
-   | Update       | Runs `freebsd-update                                          |
+   | UPDATE       | Runs `freebsd-update                                          |
    |              | <https://www.freebsd.org/cgi/man.cgi?query=freebsd-update>`__ |
    |              | to update the jail to the lateset patch level of the          |
```
Comment (wblock) [inline userguide/jails.rst]: Typo.

File: userguide/jails.rst [hunk 9]
```diff
@@ -817,44 +772,45 @@ available for a jail.
...
    +--------------+---------------------------------------------------------------+
-   | Shell        | Access a *root* command prompt to interact with a jail        |
+   | SHELL        | Access a *root* command prompt to interact with a jail        |
    |              | directly from the command line. Type :command:`exit` to       |
    |              | leave the command prompt.                                     |
    |              |                                                               |
    +--------------+---------------------------------------------------------------+
-   | Delete       | Delete the jail, all of the jail's contents, and all          |
+   | DELETE       | Delete the jail, all of the jail's contents, and all          |
    |              | associated :ref:`Snapshots`. Back up the jail's data,         |
```
Comment (wblock) [inline userguide/jails.rst]: Rephrase so the first word is not a verb that can be read as imperative:
`` Caution: deleting the jail also deletes all of the jail contents and all associated :ref:`snapshots <Snapshots>`. Back up the jail data... ``

File: userguide/jails.rst [hunk 10]
```diff
@@ -901,12 +857,12 @@ The ssh daemon
 must be enabled in a jail to allow SSH access to that jail from another
 system.
 
-The jail :guilabel:`Status` must be up before the :guilabel:`Shell`
+The jail :guilabel:`State` must be *up* before the :guilabel:`SHELL`
```
Comment (wblock) [inline userguide/jails.rst]: The "state" guilabel is all-caps above.

File: userguide/jails.rst (hunk 6 above)
Comment (a-stjohn) [reply userguide/jails.rst]: Quick Google search reveals it being used as two words.

Comment (wblock) [reply userguide/jails.rst]: Interesting, but misses the point.  We need to be consistent about this within our software and documentation.

Comment (a-stjohn) [reply userguide/jails.rst]: Majority of our docs are using "mount points".

**Pull request merged**